src/app/core/models/user/user.interface.ts

```diff
@@ -24,4 +24,6 @@ export interface UserData {
 
   businessPartnerNo?: string;
   department?: string;
+
+  active?: boolean;
```

Comment:
This property arrives only in B2B and actually does not carry additional information for the current user (because it has to be active there). Can we instead use proper B2bUser Data types? I'll commit a fixup as suggestion, please remove it if you don't agree 😉